_projects/0600_ditap.md

```diff
@@ -0,0 +1,50 @@
+---
+# Page template info (DO NOT EDIT)
+layout: default
+banner_file: banner--people-projects-lg.svg
+banner_file_mobile: banner--people-projects-sm.svg
+project_page: true
+
+# Carousel (Edit this)
+carousel_title: "Improving how the government buys technology"
+carousel_summary: "As the pace of innovation has accelerated, the government’s ability to meet people’s expectations of functional digital solutions has not kept pace. To meet this demand, we created an immersive development and training program to improve digital services acquisition expertise across government agencies.
+"
+carousel_image_name: project-ditap-card.jpg
+
+# Project detail page (Edit this)
+title: "Digital IT Acquisition Professional Program (DITAP)"
+agency: Cross-agency
+permalink: projects/ditap
+project_url: 
+
+# Impact statement (Edit this)
+impact_statement:
+  - figure: "400"
+    unit: "+"
+    description: |-
+      People will have graduated from our DITAP program by end of FY 2020
+
+  - figure: 4
+    unit: "x"
+    description: |-
+      Growth of the Digital Acquisition community since 2016, demonstrating the scale that USDS and OFPP envisioned when the program was created
+
+---
+
+## The Challenge
+
+Current federal procurement practices rarely provide the flexibility required to buy and deliver modern digital services. Government’s use of digital solutions has not kept pace with peoples’ expectations. As the pace of innovation has accelerated, the government’s ability to meet people’s expectations of functional digital solutions has not kept pace. To meet this demand, the Office of Federal Procurement Policy (OFPP) and USDS needed an immersive development and training program to improve digital services acquisition expertise across government agencies.
```

Comment:
the pace of people's expectations is called out in sentences 2 and 3; should we combine? Maybe we could remove sentence 2?